docs/docusaurus/versioned_docs/version-1.7.0/basics/quick_start_guide.md

```diff
@@ -36,11 +36,12 @@ any production hardware on hand to test an end-to-end setup.
 We'll be setting up the LTE AGW VM in this tab.
```

Comment:
My understanding is that this should not be added to the 1.7 documentation. 1.7 does not have this change. I assume the change will not be back ported to 1.7, right?